Sources/MessagingInApp/Gist/Managers/Models/Message.swift

```diff
@@ -18,7 +18,7 @@ public class GistProperties {
 
 public class Message {
     public private(set) var instanceId = UUID().uuidString.lowercased()
-    public let queueId: String?
+    public let queueId: String? // uniquely identifies an in-app message in the backend system
```

Comment:
Added this comment because[ I asked in-app squad about it](https://customerio.slack.com/archives/C036FTE6UJ0/p1715967637970009). I wasn't sure if this impacts the messages we return in queuemanager.getInlineMessages and it doesn't.

Reply:
I think the comment is a bit confusing, it probably uniquely identifies an in-app message in the queue for this user. 

Message-id is the unique identifier for the in-app message.

Reply:
[I was told that the `queueId` is a unique identifier for a message](https://customerio.slack.com/archives/C036FTE6UJ0/p1715967782604259?thread_ts=1715967637.970009&cid=C036FTE6UJ0). 

[The `messageId` refers to templates and so it may not be unique](https://customerio.slack.com/archives/C036FTE6UJ0/p1716291837428599?thread_ts=1715967637.970009&cid=C036FTE6UJ0).

Reply:
I agree `messageId` refers to template, so we can have multiple messages with the same message id in the queue. But all of them will have a unique `queueId` in that queue and the same message when expired and added again will have a new `queueId` so instead of `backend system` I would rather put it as `queue`

Reply:
I still find the name `queueId` confusing. It makes me think that there are multiple queues, and here we are referencing one specific queue by its id.

The `messageId` referencing a template is also confusing but I would be okay that it is out of scope for this PR.

For the property referenced above we can come up with something better, maybe something like `messageInstanceId` or anything else that associates the uniqueness and id directly to the message not the queue